v4.13/graph.yaml

```diff
@@ -4,7 +4,11 @@ defaultChannel: stable
 ---
 entries:
   - name: rhtas-operator.v1.1.0
-name: stable-v1.1
+    replaces: rhtas-operator.v1.0.0
+    skips:
+      - rhtas-operator.v1.0.1
+      - rhtas-operator.v1.0.2
```

Comment:
```suggestion
```

As 1.1.0 is the only version in the candidate channel we dont need to specify skips and replaces